.flowconfig

```diff
@@ -0,0 +1,24 @@
+[ignore]
```

Comment:
Get rid of all sections that are empty, and make sure each section has an empty line at the end